.github/check-executable-files.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#
+# This file is part of the phpBB Forum Software package.
+#
+# @copyright (c) phpBB Limited <https://www.phpbb.com>
+# @license GNU General Public License, version 2 (GPL-2.0)
+#
+# For full copyright and license information, please see
+# the docs/CREDITS.txt file.
+#
+set -e
+
+root="$1"
+path="${root}phpBB/"
+
```

Comment:
One of these files for extensions to use could be helpful. Like, `ext-check-executable-files.sh`. Can be the same as this file, I think just needs:
```
root="$1"
extname="$2"
path="${root}phpBB/ext/${extname}/
```